ss3_glossaries.tex

```diff
@@ -0,0 +1,31 @@
+\newacronym{abc}{ABC}{acceptable biological catch}
+\newacronym{acl}{ACL}{annual catch limit}
+\newacronym{admb}{ADMB}{AD Model Builder}
+\newacronym{alk}{ALK}{age-length-key}
+\newacronym{capam}{CAPAM}{Center for the Advancement of Population Assessment Methodology}
+\newacronym{casal}{CASAL}{C++ Algorithmic Stock Assessment Laboratory}
+\newacronym{cpue}{CPUE}{catch per unit effort}
+\newacronym{cv}{CV}{coefficient of variation}
+\newacronym{dbsra}{DB-SRA}{Depletion-Based Stock Reduction Analysis}
+\newacronym{df}{DF}{degrees of freedom}
+\newacronym{mcmc}{MCMC}{Markov chain Monte Carlo}
+\newacronym{mle}{MLE}{maximum likelihood estimation}
+\newacronym{mey}{MEY}{maximum economic yield}
+\newacronym{mpd}{MPD}{maximum posterior density}
+\newacronym{msy}{MSY}{maximum sustainable yield}
+\newacronym{nmfs}{NMFS}{National Marine Fisheries Service}
+\newacronym{noaa}{NOAA}{National Oceanic \& Atmospheric Administration}
+\newacronym{ofl}{OFL}{overfishing limit}
+\newacronym{rmse}{RMSE}{Root Mean Square Error}
+\newacronym{sb}{SB}{spawning biomass}
+\newacronym{ssb}{SSB}{spawning stock biomass}
+\newacronym{sd}{SD}{standard deviation}
+\newacronym{se}{SE}{standard error}
+\newacronym{sis}{SIS}{Species Information System}
+\newacronym{spr}{SPR}{spawning potential ratio}
+\newacronym{srr}{SRR}{stock recruitment relationship}
```

Comment:
Not sure if this should be "spawner--recruitment relationship" or "stock--recruitment relationship" but whichever is used should have an en-dash because it is a "comparative" statement.